ocs_ci/ocs/elasticsearch.py

```diff
@@ -245,3 +306,38 @@ def _copy(self, es):
             for doc in result["hits"]["hits"]:
                 log.debug(f"Going to write : {doc}")
                 es.index(index=ind, doc_type="_doc", body=doc["_source"])
+
+    def dump(self, target_path):
+        """
+        Dump All data from the internal ES server to .tgz file.
+
+        Args:
+            target_path (str): the path where the results file will be copy into
+
+        Return:
+            bool: True if the dump operation succeed and return the results data to the host
+                  otherwise False
+        """
+
+        log.info("dumping data from ES server to .tgz file")
+        rsh_cmd = f"rsh {self.dump_pod} /elasticsearch-dump/esdumper.py --ip {self.get_ip()} --port {self.get_port()}"
+        result = self.ocp.exec_oc_cmd(rsh_cmd, out_yaml_format=False, timeout=1200)
+        if "ES dump is done." not in result:
+            log.error("There is no data in the Elasticsearch server")
+            return False
+        else:
+            src_file = result.split()[-1]
+            log.info(f"Copy {src_file} from the client pod")
+
+            cp_command = f"cp {self.dump_pod}:{src_file} {target_path}/FullResults.tgz"
+            result = self.ocp.exec_oc_cmd(cp_command, timeout=120)
+            log.info(f"The output from the POD is {result}")
+            log.info("Extracting the FullResults.tgz file")
+            kwargs = {"cwd": target_path}
+            results = run_command(f"tar zxvf {target_path}/FullResults.tgz", **kwargs)
+            log.warning(f"The untar results is {results}")
```

Comment:
why is this a warning?

Reply:
since i want the the dissension on asserting or failed will be in the response of the test, and not on this function.